source/socio_econ/beh_change.rst

```diff
@@ -2,15 +2,15 @@
 
-The methodology for modeling energy choices in the residential sector using this model is described in detail in Ekholm et al. (2010) :cite:`ekholm_determinants_2010` and in the Supplementary Materials section of Pachauri et al. (2013) :cite:`pachauri_pathways_2013`. In addition to energy prices, technology costs and performance parameters, and income level of a household determining the least-cost energy-equipment combination that meets a specific energy need, two additional parameters determine choices in the model. The first is referred to as the “inconvenience cost”. An inconvenience cost is a cost related to the inconveniences associated with obtaining and using certain types of fuels. For example, gathering firewood involves an opportunity cost for the time spent in collecting it and a dis-utility to users from exposure to the smoke they inhale when it is combusted. This non-monetary cost is captured by estimating an inconvenience cost (see Ekholm et al. (2010) :cite:`ekholm_determinants_2010` for further details regarding the methodology) for each household group and fuel. This is considered an additional cost that must be taken into account by the household in making a decision regarding the choice of fuels. The second parameter that also determines energy choices for households is income dependent implicit discount rates that determine the annualized capital costs of equipment depending on their individual lifetimes.
+The methodology for modeling energy choices in the residential sector of this model is described in detail in Ekholm et al. (2010) :cite:`ekholm_determinants_2010` and in the Supplementary Materials section of Pachauri et al. (2013) :cite:`pachauri_pathways_2013`. In addition to energy prices, technology costs and performance parameters, and income level of a household determining the least-cost energy-equipment combination that meets a specific energy need, two additional parameters determine choices in the model. The first is referred to as the “inconvenience cost”. An inconvenience cost is a cost related to the inconveniences associated with obtaining and using certain types of fuels. For example, gathering firewood involves an opportunity cost for the time spent in collecting it and a dis-utility to users from exposure to the smoke they inhale when it is combusted. This non-monetary cost is captured by estimating an inconvenience cost (see Ekholm et al. (2010) :cite:`ekholm_determinants_2010` for further details regarding the methodology) for each household group and fuel. This is considered an additional cost that must be taken into account by the household in making a decision regarding the choice of fuels. The second parameter that also determines energy choices for households is income dependent implicit discount rates that determine the annualized capital costs of equipment depending on their individual lifetimes.
 
-.. image:: /_static/MESSAGE-Access_groups.png
+.. image:: /_static/MESSAGEix-Access_groups.png
```

Comment:
I don't see this image --> the image itself is still called "MESSAGE-Access_groups.png"